fetcher/fetch.go

```diff
@@ -167,9 +169,9 @@ func doReq(ctx context.Context, url string, opts GetOptions) (*http.Response, er
 	if opts.Cookie != nil {
 		req.AddCookie(opts.Cookie)
 	}
-	if opts.HeaderKeyName != "" && opts.UserKey != "" && opts.HeaderUserName != "" && opts.UserName != "" {
-		req.Header.Set(opts.HeaderKeyName, opts.UserKey)
-		req.Header.Set(opts.HeaderUserName, opts.UserName)
+	if opts.UserKey != "" && opts.UserName != "" {
+		req.Header.Set("Evergreen-Api-Key", opts.UserKey)
```

Comment:
What are these headers used for? If they're going into cedar to be forwarded to evergreen, I don't see this magic string anywhere in cedar.

Reply:
Never mind, I see the other PR in cedar for this. It's kind of weird that timber doesn't vendor cedar in any way but still has a hard dependency on it.

Reply:
Yes, it is a client for cedar (so we do not want to vendor cedar directly). But in any case, the API headers are just something that "have to be known".